releasenotes/notes/fix-codefresh-integration-563b8075e71881e4.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    CI visibility: Adds BITBUCKET_GIT_HTTP_ORIGIN environment variable
```

Comment:
Can we reword this release note. It's not immediately clear that this PR adds support for tracking the repository url via BITBUCKET_GIT_HTTP_ORIGIN environment variable.

Reply:
Done, good catch!